be/src/exec/pipeline/aggregate/spillable_aggregate_blocking_sink_operator.cpp

```diff
@@ -40,6 +40,7 @@ bool SpillableAggregateBlockingSinkOperator::is_finished() const {
 
 Status SpillableAggregateBlockingSinkOperator::set_finishing(RuntimeState* state) {
     if (_spill_strategy == spill::SpillStrategy::NO_SPILL) {
+        _is_finished = true;
```

Comment:
_is_finished won't be used under NO_SPILL mode, I think it's not necessary to set it here

Reply:
in ASAN mode mark_need_spilled may be called after set_finishing. which will cause case hang